### ISSUE TITLE
# of Batch

### ISSUE THREAD

**Issue by wjun0830** [HobbitLong/SupContrast#80]
Hi. Thanks for your great research and implementation.
However, I am not sure how to use this loss function in a multi-batch situation.
I mean, instead of using 2 batches, I am trying to use 4 batches.

Is it right that I just pass torch.cat([b1, b2, b3, b4, b5])? with the class label for b1?

Thanks a lot.

**Comment by HobbitLong** [comment]
Suppose your feature `b1` is of shape `[n, feat_dim]`, you should stack `b1,b2,b3,b4,b5` to make the concatenation like `[n, 5, feat_dim]`, for labels, it should be of shape `[n]`.

**Comment by wjun0830** [comment]
Thank you!
